analytics_extension.go

```diff
@@ -0,0 +1,45 @@
+package helix
+
+// ExtensionAnalytic ...
+type ExtensionAnalytic struct {
+	ExtensionID string `json:"extension_id"`
+	URL    string `json:"url"`
+}
+
+// ManyExtensionAnalytics ...
+type ManyExtensionAnalytics struct {
+	ExtensionAnalytics []ExtensionAnalytic `json:"data"`
+}
+
+// ExtensionAnalyticsResponse ...
+type ExtensionAnalyticsResponse struct {
+	ResponseCommon
+	Data ManyExtensionAnalytics
+}
+
+type extensionAnalyticsParams struct {
```

Comment:
This struct should be exported (ie. its start with a capital letter). It's also missing the following fields: `first`, `after`, `started_at`, `ended_at` and `type`.